yjit/src/asm/x86_64/tests.rs

```diff
@@ -1,383 +1,286 @@
 #[cfg(test)]
 
 use crate::asm::x86_64::*;
+use std::fmt;
 
-#[test]
-fn test_ret() {
-    let mut cb = CodeBlock::new();
+impl<'a> fmt::LowerHex for super::CodeBlock {
+    fn fmt(&self, fmtr: &mut fmt::Formatter) -> fmt::Result {
+        for byte in 0..self.write_pos {
+            fmtr.write_fmt(format_args!("{:02x}", self.mem_block[byte]))?;
+        }
+        Ok(())
+    }
+}
```

Comment:
I'd like to see a comment explaining what this is doing because this could be pretty cryptic for new Rust users (i.e. new team members).